kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue

```diff
@@ -103,29 +102,29 @@
         if (this.filter.value === 'allLessons') {
           return this.coachString('lessonListEmptyState');
         }
-        // if (this.filter.value === 'activeLessons') {
-        //   return this.$tr('noActiveLessons');
-        // }
-        // if (this.filter.value === 'inactiveLessons') {
-        //   return this.$tr('noInactiveLessons');
-        // }
+        if (this.filter.value === 'activeLessons') {
```

Comment:
There should be no filters for active/inactive lessons or quizzes.

Reply:
Apologies on the confusion, the tranlation strings have been refactored to visible and not visible. This are messages displayed incase the filter returns no rows.